python/cuml/ensemble/randomforestclassifier.pyx

```diff
@@ -393,9 +395,18 @@ class RandomForestClassifier(Base):
                              " please read the documentation")
 
     def _obtain_treelite_handle(self):
+        """Returns a handle to a treelite-formatted version of the model.
+        This will create a new treelite model if necessary, or return
+        a cached version when available. The handle is cached in the
+        instanced and freed at instance deletion. Caller should not
+        delete the returned model."""
         cdef ModelHandle cuml_model_ptr = NULL
         cdef RandomForestMetaData[float, int] *rf_forest = \
             <RandomForestMetaData[float, int]*><uintptr_t> self.rf_forest
+
+        if self.treelite_handle is not None:
```

Comment:
confused why the if statement is after the cdef variables are created? I think it would be nice to make the if statement as the first line of `_obtain_treelite_handle` function